Single-Model-Deployment/Batch-Inference/Batch-Transform-xgboost-high-level-SDK.ipynb

```diff
@@ -294,21 +296,6 @@
     "model_name"
    ]
   },
-  {
```

Comment:
I don't see that the SageMaker Model is created.


# Create a model
from sagemaker.model import Model
from sagemaker.predictor import Predictor

model_name = resource_name.format("Model", datetime.now().strftime("%Y-%m-%d-%H-%M-%S"))

xgb_model = Model(
    name=model_name,
    image_uri=container,
    model_data=model_url,
    role=sm_role
)
model_name


# Deploy the model to SageMaker
instance_type="ml.m5.4xlarge"

xgb_model.create()